testutil/helpers.go

```diff
@@ -9,10 +9,22 @@ import (
 	"testing"
 )
 
-var rootEnabled bool
+var (
+	rootEnabled bool
+	runParallel bool
+)
 
 func init() {
 	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
+	flag.BoolVar(&runParallel, "test.concurrent", false, "run elligible tests concurrently")
```

Comment:
It messes up the output (i.e. it's hard to read it as the test being sequential). But happy to default to that